seatunnel-connectors-v2/connector-file/connector-file-oss/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/oss/sink/OssFileSink.java

```diff
@@ -18,20 +18,24 @@
 package org.apache.seatunnel.connectors.seatunnel.file.oss.sink;
 
 import org.apache.seatunnel.api.configuration.ReadonlyConfig;
+import org.apache.seatunnel.api.sink.SupportSaveMode;
 import org.apache.seatunnel.api.table.catalog.CatalogTable;
 import org.apache.seatunnel.connectors.seatunnel.file.config.FileSystemType;
 import org.apache.seatunnel.connectors.seatunnel.file.oss.config.OssHadoopConf;
 import org.apache.seatunnel.connectors.seatunnel.file.sink.BaseMultipleTableFileSink;
 
 import java.util.Optional;
 
-public class OssFileSink extends BaseMultipleTableFileSink {
+public class OssFileSink extends BaseMultipleTableFileSink implements SupportSaveMode {
 
     private final CatalogTable catalogTable;
 
+    private final ReadonlyConfig readonlyConfig;
+
     public OssFileSink(ReadonlyConfig readonlyConfig, CatalogTable catalogTable) {
         super(OssHadoopConf.buildWithConfig(readonlyConfig), readonlyConfig, catalogTable);
         this.catalogTable = catalogTable;
+        this.readonlyConfig = readonlyConfig;
```

Comment:
```suggestion
```

Reply:
Sorry, I didn't see the difference...I've updated the code based on your suggestions. Could you please take another look and let me know if there's anything else that needs to be improved? I really appreciate your help.